modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -17,3 +17,5 @@ In addition, Sensor is responsible for all cluster interactions, such as applyin
 It collects container runtime and network activity information and sends the collected data to Sensor.
 * *Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
 However, on {ocp}, {product-title-managed-short} installs a Scanner-slim version on each secured cluster to scan images in the {ocp} integrated registry and optionally other registries.
+* *Scanner V4 DB*: This component is installed if Scanner V4 is enabled. This database stores information for Scanner V4, including index reports. For best performance, configure a persistent volume claim (PVC) for Scanner V4 DB.
```

Comment:
I always wonder if we should point out this edge-case or not. I am also fine with leaving it out: If secured-cluster-services are installed on the same cluster as central-services **in the same namespace** secured-cluster-services won't *deploy* Scanner V4 components, even though it is *enabled* (in the sense that `scanner.disable=false` and `scannerV4.disable=false`).

The semantics here differ slightly between the old config for StackRox Scanner and Scanner V4. For StackRox Scanner the `scanner.disable` switch really controls *deployment* for scanner V4 it is *enabling* (think: Auto-Sensing, in operator terminology). In the usual case it also means deploying, but in the edge case I sketched above, these two things are different.

So much for the background... not sure how much we should be going into these subtle details here.

Reply:
We could consider adding a note after the list of components:

NOTE
====
 When `secured-cluster-services` are installed on the same cluster as `central-services` and installed in the same namespace, secured-cluster-services do not deploy Scanner V4 components.

Reply:
maybe plus

> , instead it is assumed that central-services already includes a deployment of Scanner V4.